src/screens/DomainScreen/elements/RenderItem.js

```diff
@@ -293,13 +293,13 @@ const styles = StyleSheet.create({
     fontSize: normalizeFontSize(16),
     fontFamily: fonts.inter[700],
     lineHeight: normalizeFontSize(24),
-    color: COLORS.white2
+    color: COLORS.white
   },
   domainItemDescription: {
     fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(16),
     lineHeight: normalizeFontSize(24),
-    color: COLORS.white2,
+    color: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text in `domainItemTitle` and `domainItemDescription` has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change doesn't affect readability or contrast with the background, especially considering accessibility guidelines for color contrast. If `COLORS.white` is less visible than `COLORS.white2`, it might be better to revert this change.
```diff
-    color: COLORS.white2
+    color: COLORS.white
```
If `COLORS.white` provides better visibility and meets accessibility standards, then this change is good to go.


<!-- This is an auto-generated comment by OSS CodeRabbit -->